test/helpers/holiday_helpers.rb

```diff
@@ -0,0 +1,46 @@
+module HolidayHelpers
+  class MissingConfigurationError < RuntimeError; end
+
+  HOLIDAYS = {
+    ups: { # https://compass.ups.com/ups-holiday-schedule-2016/
+      "2016" => [
+        [1,  1 ],
```

Comment:
suggestion: Change this to a hash with `month` and `day` keys? 

I don't feel strongly about this suggestion but could be more readable.

Also, having such a complex data structure makes me feel like there should be a `Holidays` object or something, but maybe thats an overkill for these tests.